src/SfxWeb/src/app/modules/backup-restore/storage-form/storage-form.component.ts

```diff
@@ -71,10 +84,43 @@ export class StorageFormComponent implements OnInit {
 
       storage.get('Path').setValidators([Validators.required]);
     }
-
+    this.updateStorageKindValidatorsPrimaryCredentials(storage, false);
+    this.updateStorageKindValidatorsSecondaryCredentials(storage, false);
     storage.get('ContainerName').updateValueAndValidity();
     storage.get('ConnectionString').updateValueAndValidity();
     storage.get('Path').updateValueAndValidity();
   }
+  updateStorageKindValidatorsPrimaryCredentials(storage: AbstractControl, IsEmptyPrimaryCredential: boolean){
+    if(IsEmptyPrimaryCredential === true)
```

Comment:
remove === true to simplify this.

Reply:
Thanks. I have made the necessary changes